### PR TITLE
use pushd and popd to enable bat build from different dir #2215

### PR DESCRIPTION
**Pull request by TimKam**

### CODE REVIEW

**Comment by tk0miya** [comment]
Could you check EOL code of your fix?
`\r\n` is expected for `make.bat`.
https://travis-ci.org/sphinx-doc/sphinx/jobs/169900390#L1407

@shimizukawa Could you check this please? I don't have windows environment...

**Comment by TimKam** [comment]
Locally on my Windows machine it's all CRLF.
I also activated `autocrfl` in my git settings, re-added the file and committed again.
Any idea?
Perhaps adding a [.gitattributes](https://help.github.com/articles/dealing-with-line-endings/#example) file that enforces CRLF  could help.

File: sphinx/templates/quickstart/make.bat.new_t [hunk 1]
```diff
@@ -31,4 +33,4 @@ goto end
 %SPHINXBUILD% -M help %SOURCEDIR% %BUILDDIR% %SPHINXOPTS%
 
 :end
-
+popd
```
Comment (shimizukawa) [inline sphinx/templates/quickstart/make.bat.new_t:36]: This last line must have CRLF. In other word, empty line is required.

File: sphinx/templates/quickstart/make.bat_t [hunk 2]
```diff
@@ -279,4 +281,4 @@ if "%1" == "dummy" (
 )
 
 :end
-
+popd
```
Comment (shimizukawa) [inline sphinx/templates/quickstart/make.bat_t:284]: Please add empty line at last.

**Comment by shimizukawa** [comment]
IMO, this is a bugfix. So, I'll fix the travis error and merge this into 1.5-release (stable doesn't have templates/make.bat*).
@tk0miya do you have any opinion?

**Comment by shimizukawa** [comment]
Why is it a bug? Because:
When some trouble were happen on `make latexpdf` process, current directory is moved into `_build/latexpdf` instead of a location before invoking make command.
This PR will fix the problem.

**Comment by shimizukawa** [comment]
merged by hand into 1.5-release branch. Thanks!

**Comment by tk0miya** [comment]
No objection for 1.5-release.
Surely, it's bug.

**Pull request closed without merging**